starters/oasp4j-web-starter/pom.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd"
+  xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>io.oasp.java.dev</groupId>
+    <artifactId>oasp4j-starters</artifactId>
+    <version>dev-SNAPSHOT</version>
+  </parent>
+
+  <groupId>io.oasp.java.starters</groupId>
+  <artifactId>oasp4j-web-starter</artifactId>
+  <version>${oasp4j.version}</version>
+  <packaging>jar</packaging>
+  <name>${project.artifactId}</name>
+  <url>http://maven.apache.org</url>
+
+  <dependencies>
+    <dependency>
+      <groupId>${oasp.modules.groupId}</groupId>
+      <artifactId>oasp4j-web</artifactId>
+      <version>${oasp4j.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>org.springframework.boot</groupId>
+      <artifactId>spring-boot-starter-web</artifactId>
+      <exclusions>
+        <exclusion>
+          <groupId>org.springframework.boot</groupId>
+          <artifactId>spring-boot-starter-tomcat</artifactId>
+        </exclusion>
+        <exclusion>
+          <groupId>org.springframework.boot</groupId>
+          <artifactId>spring-boot-starter-validation</artifactId>
+        </exclusion>
+      </exclusions>
+    </dependency>
+    <dependency>
+      <groupId>${oasp.modules.groupId}</groupId>
+      <artifactId>oasp4j-test</artifactId>
```

Comment:
has to be `<scope>test</scope>` - please fix in all places. IMHO we can drop this dependency from all starters (as long as there are no JUnits in these starters).